.github/workflows/cc-payload-after-push-amd64.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
           - cc-virtiofsd
           - cc-sev-kernel
           - cc-sev-ovmf
+          - cc-x86-ovmf
```

Comment:
Not a blocker by any means, but would it be good to specify `x86_64` vs `x86` here?